lua/lvim/plugins.lua

```diff
@@ -6,25 +6,28 @@ local core_plugins = {
     lazy = true,
     dependencies = { "mason-lspconfig.nvim", "nlsp-settings.nvim" },
   },
-  { "williamboman/mason-lspconfig.nvim", lazy = true },
+  { "williamboman/mason-lspconfig.nvim", lazy = true, dependencies = "mason.nvim" },
   { "tamago324/nlsp-settings.nvim", lazy = true },
   { "jose-elias-alvarez/null-ls.nvim", lazy = true },
   {
     "williamboman/mason.nvim",
     config = function()
       require("lvim.core.mason").setup()
     end,
+    lazy = true,
   },
```

Comment:
I can't use `:Mason` cmd now, could you please add `cmd = "Mason"` here?

Reply:
thanks for the review, I missed that, I'll also need to add LspInstall